src/main/java/com/rev_connect_api/models/Post.java

```diff
@@ -42,6 +45,9 @@ public class Post {
     @Column(name = "updated_at", nullable = false)
     private LocalDateTime updatedAt;
 
+    @LastModifiedDate
```

Comment:
Not sure if this will give accurate dates. @LastModified date can be used for updated_at because it is the column for last modified date. The pinned_at timestamp should only be for when a post is pinned. This will probably be done manually.